parcels/grid.py

```diff
@@ -3,12 +3,13 @@
 from ctypes import Structure, c_int, c_float, c_double, POINTER, cast, c_void_p, pointer
 from enum import IntEnum
 
-__all__ = ['GridCode', 'StructuredGrid', 'GridIndex', 'CGrid']
+__all__ = ['GridCode', 'RectilinearGrid', 'RectilinearSGrid', 'GridIndex', 'CGrid']
 
 
 class GridCode(IntEnum):
-    StructuredGrid = 0
-    SemiStructuredGrid = 1
+    RectilinearGrid = 0
+    RectilinearSGrid = 1
+    CurvilinearGrid = 2
```

Comment:
Already implement both `CurvilinearZGrid` and `CurvilinearSGrid`?

Reply:
No, it is not yet done. It will be done in next PR.
(I've just changed the SemiStructuredGrid line to the new names, even not available)